contracts/mocks/EnumerableSetMock.sol

```diff
@@ -31,7 +31,7 @@ contract EnumerableSetMock{
         return set.length();
     }
 
-    function get(uint256 index) public view returns (address) {
```

Comment:
I didn't understand the reasoning behind this renaming. Can you explain some more?

Reply:
See [this comment](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2151#issuecomment-605077802).